layouts/index.html

```diff
@@ -48,6 +48,18 @@ <h3>{{ $.Param "usecases.mes_aides.title" }}</h3>
 			</figure>
 		</section>
 
+		<section class="row">
+			<div class="left-col">
+				<h3>{{ $.Param "usecases.leximpact.title" }}</h3>
+				<p>{{ $.Param "usecases.leximpact.description" }}</p>
+			</div>
+
+			<figure>
+				<img class="screenshot" src="/img/leximpact.png" alt="LexImpact">
```

Comment:
Change the image file with [this one](https://user-images.githubusercontent.com/17675313/68288289-6a75a780-0084-11ea-9bab-addb613cd2a4.png)? In this screenshot, the top of the header is missing. 🙂

Reply:
Thanks for the heads up.

Ratio must be 1280x720, yours is 1897x941.

Could you provide an alternative image with the good ratio?

I'll change it then :)

Reply:
I'll provide a new one, you let me know what do you think :)

Reply:
Done!

Reply:
Done!